databases/de-database-schema/src/main/conversions/v2.8.0/c280_2016071201.clj

```diff
@@ -0,0 +1,39 @@
+(ns facepalm.c280-2016071201
+  (:use [kameleon.sql-reader :only [exec-sql-statement load-sql-file]])
+  (:require [korma.core :as sql]))
+
+(def ^:private version
+  "The destination database version"
+  "2.8.0:20160712.01")
+
+(defn- add-user-id-column
+  "Adds a user_id column to the integration_data table."
+  []
+  (println "\t* adding the user_id column to the integration_data table.")
+  (->> ["ALTER TABLE ONLY integration_data ADD COLUMN user_id uuid"
+        "ALTER TABLE ONLY integration_data ADD CONSTRAINT integration_data_user_id_fk
+         FOREIGN KEY (user_id) REFERENCES users (id)"]
```

Comment:
Yeah, that makes sense. I'm not entirely sure why I didn't use `load-sql-file` here, especially since I used it elsewhere in this PR.